src/CreditCard.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable jsx-a11y/alt-text */
+/* eslint-disable react/jsx-no-comment-textnodes */
+import React, { Component } from 'react';
+
+class CreditCard extends Component {
+  render() {
+    return (
+      <div className="CreditCard" style={{
+        backgroundColor: this.props.bgColor,
+        color: this.props.color,
+      }}>
+        <div className="type">
+          // eslint-disable-next-line jsx-a11y/alt-text
```

Comment:
this doesn't work in html, you have to do it like this: {/* eslint-disable-next-line jsx-a11y/alt-text */}

By the way, your styling is over the top!